.gitmodules

```diff
@@ -16,3 +16,6 @@
 [submodule "relays"]
 	path = relays
 	url = https://github.com/keep-network/relays.git
+[submodule "bitcoin/electrs/electrs"]
+	path = bitcoin/electrs/electrs
+	url = https://github.com/Blockstream/electrs
```

Comment:
Why submodule this instead of just cloning it directly in the dockerfile? It's not really part of our development process, so the latter seems cleaner.